packages/bottender-luis/src/index.ts

```diff
@@ -52,6 +53,21 @@ module.exports = function luis({
   bingSpellCheckSubscriptionKey?: string;
   log?: boolean;
 }): Action<Context<any, any>> {
+  invariant(
+    typeof appId === 'string' && appId.length > 0,
+    'App ID is required parameter.'
```

Comment:
'luis: `appId ` is required.'

or 

'luis: `appId ` is a required parameter.'